packages/plugin-react/src/index.ts

```diff
@@ -72,7 +72,11 @@ declare module 'vite' {
       /**
        * Manipulate the Babel options of `@vitejs/plugin-react`
        */
-      reactBabel?: (options: ReactBabelOptions, config: ResolvedConfig) => void
+      reactBabel?: (
+        options: ReactBabelOptions,
+        // eslint-disable-next-line @typescript-eslint/consistent-type-imports
+        config: import('vite').ResolvedConfig
+      ) => void
```

Comment:
Is there a reason we don't add `ResolvedConfig` on this line instead?

https://github.com/vitejs/vite/blob/5818ac927861783ea2b05450761fed30f40e7399/packages/plugin-react/src/index.ts#L5

Reply:
`tsc` would complain that 'ResolvedConfig' is declared but never used. see https://github.com/vitejs/vite/runs/5358118477?check_suite_focus=true

Reply:
I see. When I tested locally, I didn't get the error, which is strange 🤔